curriculum/challenges/english/05-back-end-development-and-apis/basic-node-and-express/chain-middleware-to-create-a-time-server.md

```diff
@@ -49,7 +49,7 @@ The /now endpoint should have mounted middleware
   );
 ```
 
-The /now endpoint should return a time that is +/- 20 secs from now
+The /now endpoint should return a time that is no more than 20 secs different from now
```

Comment:
```suggestion
The /now endpoint should return the current time
```

Please use this wording as mentioned by @ShaunSHamilton in https://github.com/freeCodeCamp/freeCodeCamp/issues/46941#issuecomment-1187879000